File: app/src/main/java/com/BlizzardArmory/warcraft/Equipment/Effect.java
package com.BlizzardArmory.warcraft.Equipment;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Effect {

    @SerializedName("display_string")
    @Expose
    private String displayString;
    @SerializedName("required_count")
    @Expose
    private int requiredCount;

    public String getDisplayString() {
        return displayString;
    }

    public void setDisplayString(String displayString) {
        this.displayString = displayString;
    }

    public int getRequiredCount() {
        return requiredCount;
    }

    public void setRequiredCount(int requiredCount) {
        this.requiredCount = requiredCount;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/equipment/Effect.kt
package com.BlizzardArmory.model.warcraft.equipment

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Effect.
 */
@Keep
data class Effect(

    @SerializedName("display_string")
    var displayString: String,

    @SerializedName("required_count")
    var requiredCount: Int

)